src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers.csproj

```diff
@@ -6,6 +6,9 @@
     <VersionPrefix>0.1.1</VersionPrefix>
     <LangVersion>8</LangVersion>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <AssemblyVersion>0.1.2.0</AssemblyVersion>
```

Comment:
remove

Reply:
all the version updates, or just this one?

Reply:
All of them, CI should be doing the prefix generation.